src/All-Movies/All-Movies.css

```diff
@@ -0,0 +1,11 @@
+.movies-container {
+        display: grid;
+        grid-template-columns: 300px 300px 300px 300px 300px 300px 300px 300px;
```

Comment:
No neet to change now, but this can be refactored with `repeat(8 f1r)` where that line is saying make 8 columns that take up one fraction of the space (I am pretty sure fr is a fraction). Basically just 8 columns of the same size.